tests/integration/helpers.py

```diff
@@ -18,3 +27,62 @@ async def get_leader_unit(model, application_name: str) -> Unit:
         if unit.application == application_name and await unit.is_leader_from_status():
             return unit
     raise RuntimeError(f"Leader unit for `{application_name}` not found.")
+
+
+async def get_unit_status_messages(
+    ops_test: OpsTest, app_name: str = APP_NAME
+) -> List[tuple[str, str]]:
+    """Get the status messages from all the units of the given application."""
+    return_code, stdout, stderr = await ops_test.juju("status", "--format", "yaml", app_name)
+    if return_code:
+        raise RuntimeError(stderr)
+    output = yaml.safe_load(stdout)
+    unit_statuses = output["applications"][app_name]["units"]
+    print(unit_statuses)
```

Comment:
Was this print intentional?

Reply:
absolutely not. I missed this when carrying it over from my test branch. Thanks!

Reply:
https://github.com/canonical/vault-k8s-operator/pull/429/commits/e877ef39c6ba4eb4b2d8d5f740f2e2aafb6a92b2